.travis.yml

```diff
@@ -1,6 +1,9 @@
 sudo: false
 language: ruby
 rvm:
-  - 2.3.1
-before_install:
-  - gem install bundler
+  - 2.6.2
+  - 2.5.5
+  - 2.4.4
+  - 2.3.7
```

Comment:
since we are supporting ruby version >= 2.1.0 I suggest to add 2.2.x and 2.1.x here

Reply:
Let's move these version under allow failure like here: https://github.com/trailblazer/trailblazer-activity-dsl-linear/pull/3

What do you think?